### PR TITLE
doc: update up and running instructions

### PR DESCRIPTION
**Pull request by nazreen**
the now deleted part of the doc was the default text from the vue starter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,33 @@
...
+PROD_FIREBASE_PROJECT_ID:
+PROD_FIREBASE_STORAGE_BUCKET:
+PROD_FIREBASE_MESSAGING_SENDER_ID:
 
-# run all tests
-npm test
 ```
 
-For a detailed explanation on how things work, check out the [guide](http://vuejs-templates.github.io/webpack/) and [docs for vue-loader](http://vuejs.github.io/vue-loader).
+todo: use DEV\_ variables instead (set in config/dev.env.js)
```
Comment (weiweitoo) [inline README.md]: Just curious, is config/dev.env.js not that good?

Comment (weiweitoo) [reply README.md]: Oh. Does this TODO mean we should do it in config/dev.env.js?

Comment (nazreen) [reply README.md]: @weiweitoo I've updated the sentence to be more clear. this code is outdated. I will tag you in the latest code

File: README.md [hunk 2]
```diff
@@ -1,43 +1,33 @@
...
+PROD_FIREBASE_PROJECT_ID:
+PROD_FIREBASE_STORAGE_BUCKET:
+PROD_FIREBASE_MESSAGING_SENDER_ID:
 
-# run all tests
-npm test
 ```
 
-For a detailed explanation on how things work, check out the [guide](http://vuejs-templates.github.io/webpack/) and [docs for vue-loader](http://vuejs.github.io/vue-loader).
+todo: use `DEV_` variables when in dev mode (set this in config/dev.env.js). Currently all the variables names are `PROD_`
```
Comment (nazreen) [inline README.md]: @weiweitoo actually right now I'm a bit confused myself lol. our current implementation is using .env whereas vue by default provides us the ability to have environment-specific env variables. I was trying to say we should have environment specific env variables. I need to do a revision of how this works in vue.

**Pull request merged**